src/test/java/org/folio/rest/api/RequestsApiTest.java

```diff
@@ -926,6 +931,86 @@ public void cannotUpdateRequestForSameItemToAnExistingPosition()
       "Cannot have more than one request with the same position in the queue")));
   }
 
+  @Test
+  public void cannotCreateItemLevelRequestIfItemIdAndHoldingsRecordIdAreNull()
```

Comment:
Combine into one awesome test:
- values: valida value, null value, absent
- for both fields

Reply:
Resolved